cypress/integration/item_details.js

```diff
@@ -15,6 +16,15 @@ describe('Item details view', function() {
 		cy.contains(todayString);
 	});
 
+	
+	it('Does the delete item button function correctly?', function() {
```

Comment:
**No need to fix**

Generally, this should not be phrased as a question but an affirmation instead, what I do is always concatenate the `it` + `test _description_`

I think something like this could work `it should delete an item when the Delete item button is clicked` - or something between those lines.

cc @the-collab-lab/tcl-2 to give this more visibility! 👀